File: DotNetLoginService/UserLoginService/Models/Payment.cs
using System;
using System.Collections.Generic;

namespace UserLoginService.Models
{
    public partial class Payment
    {
        public int PayId { get; set; }
        public string? PayMethod { get; set; }
        public int? Bid { get; set; }
        public string? TransNo { get; set; }
        public int? Oid { get; set; }
        public decimal? Amount { get; set; }

        public virtual Buyer? BidNavigation { get; set; }
        public virtual Order? OidNavigation { get; set; }
    }
}

File: Kisan_connect/KisaanConnect/src/main/java/com/example/demo/entity/Payment.java
package com.example.demo.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.sql.Timestamp;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "payment")
public class Payment {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer payId;
    
    private String payMethod;
    private String transNo;
    private BigDecimal amount;

    @ManyToOne
    @JoinColumn(name = "bid")
    private Buyer buyer;

    @ManyToOne
    @JoinColumn(name = "oid")
    private Order order;

    // Getters and Setters
}